src/components/Login/AuthForm.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import { Stack, Typography } from "@mui/material";
+
+import { faUser, faLock, faEnvelope } from "@fortawesome/free-solid-svg-icons";
+import {
+  faFacebook,
+  faTwitter,
+  faInstagram,
+} from "@fortawesome/free-brands-svg-icons";
+import SocialIcons from "./SocialIcons/SocialIcons";
+import Input from "./Input";
+import Title from "../Texts/Title";
+import Submit from "./SubmitButton";
+const AuthForm = (props) => {
+  const[name,setName]=useState("");
+  const[email,setEmail]=useState("");
+  const[password,setPassword]=useState("");
+  function logIn(){
+    console.log("inside login ");
+  }
+ async function  signUp  (){
+    let result= await fetch("http://localhost:3006/signup",{
+      method:'post',
+      body:JSON.stringify({name,email,password}),
+      headers:{
+        'Content-Type':'application/json'}
+    });
+    result= await result.JSON();
+    console.log("inside signup ");
+  }
+  return (
+    <form
+      action=""
+      className={props.type === "signin" ? "sign-in-form" : "sign-up-form"}
+      onSubmit={(e) => {
+        e.preventDefault(); // Prevent form submission from refreshing the page
+        // Call your submit logic here
+        if (props.type === "signin") {
```

Comment:
please replace this with a ternary operator 
`{props.type === "'signin" ? logIn() : signUp()}`

Reply:
done